src/api/python/BUILD.bazel

```diff
@@ -45,7 +45,7 @@ py_wheel(
         "protobuf==5.29.3",
     ],
     strip_path_prefixes = ["src/api/python/"],
-    version = "0.8.1",
+    version = "0.8.2",
```

Comment:
might be worth a `0.9.0` bump here since the deps and proto packaging changed a reasonable amount.